go.mod

```diff
@@ -53,7 +53,6 @@ require (
 	github.com/opencontainers/selinux v1.10.0
 	github.com/pkg/errors v0.9.1
 	github.com/pmezard/go-difflib v1.0.0
-	github.com/prometheus/client_golang v1.11.1 // indirect
```

Comment:
I think @lsm5 just added this to fix up a CVE.  Should this be removed like this Lokesh?

Reply:
This is removed automatically by make vendor because the version is already specified in buildah I think.